### PR TITLE
RHDM-1049 Wrong output variable values in PMML mining model

### PR DESCRIPTION
**Pull request by lanceleverich**
* Changed output of the target field to be a simple value that is set at
the same time as the Sum_ version is created.
* Updated test to make sure change was effective in adding the output
value

### CODE REVIEW

File: kie-pmml/src/test/java/org/kie/pmml/pmml_4_2/predictive/models/mining/MiningModelSumRegressionTest.java [hunk 1]
```diff
@@ -55,8 +56,8 @@
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][] {
                 { Optional.of(10.0), Optional.of(10.0), Optional.of(10.0), 2070 },
-                { Optional.of(200.0), Optional.of(-1.0), Optional.of(2.0), -299 },
-                { Optional.of(90.0), Optional.of(2.0), Optional.of(4.0), 17040 },
+                                            {Optional.of(200.0), Optional.of(-1.0), Optional.of(2.0), -299},
```
Comment (jiripetrlik) [inline kie-pmml/src/test/java/org/kie/pmml/pmml_4_2/predictive/models/mining/MiningModelSumRegressionTest.java]: Please revert back original indent.

**Review by jiripetrlik — CHANGES_REQUESTED** [review]
Only minor comment. There is probably accidentally changed indent in one file. Otherwise looks good. But we need to wait for finishing of downstream PR check before merge.

**Comment by jiripetrlik** [comment]
jenkins execute full downstream build

File: kie-pmml/src/test/java/org/kie/pmml/pmml_4_2/predictive/models/mining/MiningModelSumRegressionTest.java (hunk 1 above)
Comment (lanceleverich) [reply kie-pmml/src/test/java/org/kie/pmml/pmml_4_2/predictive/models/mining/MiningModelSumRegressionTest.java]: Fixed

**Comment by jiripetrlik** [comment]
One more comment. I did not notice it before, but please change JIRA name in PR and in commit. It is RHDM-1049 instead of RHPAM-1049.

**Review by jiripetrlik — APPROVED** [review]

**Review by danielezonca — APPROVED** [review]

**Pull request merged**